src/app/ui/NewTaskForm.tsx

```diff
@@ -211,12 +211,20 @@ export const NewTaskForm = ({
               }}
               selectorType={SelectorType.ASSIGNEE_SELECTOR}
               buttonContent={
-                <Typography variant="bodySm" lineHeight="20px" sx={{ color: (theme) => theme.color.gray[600] }}>
-                  {truncateText(
-                    (assigneeValue as IAssigneeCombined)?.name ||
-                      `${(assigneeValue as IAssigneeCombined)?.givenName ?? ''} ${(assigneeValue as IAssigneeCombined)?.familyName ?? ''}`.trim(),
-                    TruncateMaxNumber.SELECTOR,
-                  )}
+                <Typography
+                  variant="bodySm"
+                  lineHeight="20px"
+                  sx={{
+                    color: (theme) => theme.color.gray[600],
+                  }}
+                >
+                  {assigneeValue
+                    ? truncateText(
+                        (assigneeValue as IAssigneeCombined)?.name ||
+                          `${(assigneeValue as IAssigneeCombined)?.givenName ?? ''} ${(assigneeValue as IAssigneeCombined)?.familyName ?? ''}`.trim(),
+                        TruncateMaxNumber.SELECTOR,
+                      )
+                    : 'Assignee'}
```

Comment:
All 3 buttons i.e. Status, due date, and assignee doesn't match the size in the design. According to the design, size (height) should be 30px, but here the size (height) is around 22px. Let's not give fix height, but fixing the size of the icon and giving accurate padding might correct this. 

From this PR -
![Screenshot from 2024-07-05 15-17-21](https://github.com/copilot-platforms/tasks-app/assets/38468429/7eaed701-98e9-4f2b-993a-44e26f4de21a)

From Figma design - 
![Screenshot from 2024-07-05 15-19-34](https://github.com/copilot-platforms/tasks-app/assets/38468429/67bbf860-7478-479f-84dc-b3c2a7ae525f)